_posts/2019-02-11-bootstrap-4-3-0.md

```diff
@@ -0,0 +1,58 @@
+- **Improved:** Simplified popover arrow CSS by generating arrows on the `.popover` instead of an additional element, `.popover-arrow`.
+- **Improved:** Badge focus styles now match their `background-color` like our buttons.
+- **Fixed:** Reverted v4.2.1's change to the breakpoint and grid container Sass maps that blocked folks from upgrading when modifying those default variables.
+- **Fixed:** Restored `white-space: nowrap` to `.dropdown-toggle` (before v4.2.1 it was on all `.btn`s) so carets don't wrap to new lines.
+- **Deprecated:** `img-retina`, `invisible`, `float`, and `size` mixins are now deprecated and will be removed in v5.
+
+Checkout the full [v4.3.0 ship list](https://github.com/twbs/bootstrap/issues/27893) and [GitHub project](https://github.com/twbs/bootstrap/projects/16) for the full details.
+
+[Head to to the v4.3.x docs](https://getbootstrap.com/docs/4.3/) to see the latest in action. The full release has been published to npm and will soon appear on the Bootstrap CDN and Rubygems.
+
+## Introducing responsive font sizes
+
+![Responsive font-sizes](https://camo.githubusercontent.com/eb6458d9bb840c5b5764a33ac626ca084f12e460/68747470733a2f2f692e696d6775722e636f6d2f50326978614b482e706e67)
+
+Our biggest new addition to Bootstrap in v4.3 is responsive font sizes, a [new project in the Bootstrap GitHub org](https://github.com/twbs/rfs) to automate calculate an appropriate `font-size` based on the dimensions of a visitor's device or browser viewport. Here's how it works:
+
+- All `font-size` properties have been switched to the `@include font-size()` mixin. Our Stylelint configuration now prevents the usage of `font-size` property.
+
+- Disabled by default, you can opt into this new behavior by toggling the `$enable-responsive-font-sizes` boolean variable.
+
+- The base `font-size` is set via the `$rfs-base-font-size` Sass variable. Modifying that value and recompiling will automatically modify any element's font-size when using the new mixin.
+
+We've enabled responsive font-sizes in our own docs starting with v4.3. Please share feedback with us via GItHub issues or on Twitter. We've added some light guidance to [our Typography docs](https://getbootstrap.com/docs/4.3/content/typography/#responsive-font-sizes) to explain the feature. You can also learn more by [reading the rfs project documentation](https://github.com/twbs/rfs).
+
+## Branches, Hugo, and jQuery
+Right after shipping v4.3, we'll be tackling a few key changes on our road to active v5 development. These are larger changes to how we maintain and develop Bootstrap and are considered foundational for v5.
+
+- **Improving our branches for development.** `master` will become our new `v3-dev` branch (used only for critical security fixes). `v4-dev` will stay as-is, but we'll cut a new `master` branch from there to develop v5.
```

Comment:
>`master` will become our new `v3-dev` branch (used only for critical security fixes)

Not sure about this because I don't want people to expect any change in 3.x TBH. Maybe:

>`master` will become our new `v3-dev` branch .